shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API/Wallet.hs

```diff
@@ -140,13 +139,12 @@ getNonMyopicMemberRewards globals ss creds =
         poolParams
     histLookup k = fromMaybe mempty (Map.lookup k ls)
     topPools = getTopRankedPools rPot (Coin totalStake) pp poolParams (fmap percentile' ls)
-    mkNMMRewards ms k (ap, poolp, sigma) =
+    mkNMMRewards t (hitRateEst, poolp, sigma) =
       if checkPledge poolp
-        then nonMyopicMemberRew pp poolp rPot s ms nmps ap
+        then nonMyopicMemberRew pp rPot poolp s sigma t topPools hitRateEst
         else mempty
       where
         s = (toShare . _poolPledge) poolp
-        nmps = nonMyopicStake k sigma s pp topPools
```

Comment:
the reason for this change, namely calculating the non-myopic state _inside_ of `nonMyopicMemberRew` is just for consistency with the delegation design document.